iceberg/iceberg-handler/src/test/results/positive/llap/iceberg_major_compaction_partition_evolution.q.out

```diff
@@ -379,3 +446,19 @@ CompactionId	Database	Table	Partition	Type	State	Worker host	Worker	Enqueue Time
 #Masked#	default	ice_orc	company_id=100/dept_id=1	MAJOR	succeeded	#Masked#	manual	iceberg	0	0	0	 --- 
 #Masked#	default	ice_orc	company_id=100/dept_id=2	MAJOR	succeeded	#Masked#	manual	iceberg	0	0	0	 --- 
 #Masked#	default	ice_orc	 --- 	MAJOR	succeeded	#Masked#	manual	iceberg	0	0	0	 --- 
+PREHOOK: query: select `partition`, spec_id, content, record_count
+from default.ice_orc.files
+order by `partition`, spec_id, content, record_count
+PREHOOK: type: QUERY
+PREHOOK: Input: default@ice_orc
+#### A masked pattern was here ####
+POSTHOOK: query: select `partition`, spec_id, content, record_count
+from default.ice_orc.files
+order by `partition`, spec_id, content, record_count
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@ice_orc
+#### A masked pattern was here ####
+{"company_id":100,"dept_id":1,"team_id":null}	1	0	3
+{"company_id":100,"dept_id":2,"team_id":null}	1	0	3
+{"company_id":null,"dept_id":null,"team_id":null}	1	0	1
+{"company_id":null,"dept_id":null,"team_id":null}	1	0	2
```

Comment:
I wonder why two `{"company_id":null,"dept_id":null,"team_id":null}` remain. I think it is minor and out of scope of HIVE-28727